website/snippets/_cloud-environments-info.md

```diff
@@ -28,11 +28,33 @@ Both development and deployment environments have a section called **General Set
 | Default to Custom Branch | ☑️ | Determines whether to use a branch other than the repository’s default  | See below |
 | Custom Branch | dev | Custom Branch name | See below |
 
-:::note About dbt version
+### About versions in dbt Cloud
 
-- dbt Cloud allows users to select any dbt release. At this time, **environments must use a dbt version greater than or equal to v1.0.0;** [lower versions are no longer supported](/docs/dbt-versions/upgrade-core-in-cloud).
-- If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
+- Multiple dbt releases: dbt Cloud enables users to choose from various dbt versions.
+- Version requirements: Environments must use dbt version v1.0.0 or higher. [Lower versions are not supported](/docs/dbt-versions/upgrade-core-in-cloud).
+- Automatic updates: Selecting a version labeled `(latest)` ensures your environment automatically installs the latest stable release of that minor version selected.
+
+
+### Git repository caching 
```

Comment:
this section is duplicated. 

list is ordered alphabetically, should be after "Extended attributes" section.

here's the page TOC from [vercel preview](https://docs-getdbt-com-git-notices-dbt-labs.vercel.app/docs/deploy/deploy-environments) :

![Screenshot 2024-01-09 at 2 11 23 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/55f7f015-0948-49ee-856b-840672e12db6)

Reply:
Going to revert this whole file. Something funky happened with the additions here that weren't supposed to.